test/e2e/runner/load.go

```diff
@@ -202,3 +202,7 @@ func loadProcess(ctx context.Context, testnet *e2e.Testnet, chTx <-chan types.Tx
 		}
 	}
 }
+
+func loadSingleNode(ctx context.Context, rpcEndpoint string, chTx <-chan types.Tx) {
```

Comment:
Was there meant to be something more here?

Reply:
good catch. That was part of some other work I was doing but I guess I was on the wrong branch